src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10635,3 +10677,123 @@ definitions:
+      entry:
+        $ref: '#/definitions/Rule'
+  Rule:
+    type: object
+    required:
+      - id
+      - name
+      - conditions
+      - actions
+    properties:
+      id:
+        description: Identifier for the rule
+        type: string
+      name:
+        description: Name of the rule
+        type: string
+      description:
+        description: Description of the rule
+        type: string
+      enabled:
+        description: Whether the rule is enabled
+        type: boolean
+        default: true
+      cascade:
+        description: Whether the rule also applies to subfolders
+        type: boolean
+        default: false
+      asynchronous:
+        description: Whether the rule should be run in the background
+        type: boolean
+        default: false
+      errorScript:
+        description: If the rule should be run in the background then an optional error script can be referenced
+        type: string
+      trigger:
+        description: |
+          The trigger(s) that cause the rule to be activated.
+          * inbound - The rule should be activated when an item enters the folder
+          * update - The rule should be activated when an item within the folder is updated
+          * outbound - The rule should be activated when an item leaves the folder or is deleted
+        type: array
+        items:
+          type: string
+          enum:
+          - inbound
+          - update
+          - outbound
+        default: ["inbound"]
+      conditions:
+        description: The condition that determines whether the actions will be executed for a rule
+        type: '#/definitions/CompositeConditionDefinition'
+      actions:
+        description: The actions for the rule
+        type: array
+        items:
+          $ref: '#/definitions/ActionDefinition'
+  CompositeConditionDefinition:
+    type: object
+    properties:
+      inverted:
+        description: Whether to invert the logic for this condition
+        type: boolean
+        default: false
+      booleanMode:
+        description: How to combine the clauses of this condition ("and" or "or")
+        type: string
+        enum:
+        - and
+        - or
+        default: "and"
+      compositeConditions:
+        description: Nested list of composite clauses in this condition
+        type: array
+        items:
+          $ref: '#/definitions/CompositeConditionDefinition'
+        default: []
+      simpleConditions:
+        description: Nested list of simple (per field) conditions.
+        type: array
+        items:
+          $ref: '#/definitions/SimpleConditionDefinition'
+        default: []
+  SimpleConditionDefinition:
+    type: object
+    required:
+      - field
+      - comparator
+      - parameter
+    properties:
+      field:
+        description: The name of the property to compare (e.g. "cm:name")
+        type: string
+      comparator:
+        description: The method to compare the property against the parameter
```

Comment:
Should we give a list of possible values (or a way to discover them, e.g., new endpoint etc.)?